Source/Components/ClusterLayout.swift

```diff
@@ -14,5 +14,17 @@ import Foundation
 ///
 /// See the documentation of the clusters for more informations and configuration options.
 public enum ClusterLayout {
-    /* empty ATM, extended in other files... at some point there will be something here */
+    public enum Orientation: String {
+        case horizontal
+        case vertical
+    }
```

Comment:
I am not sure if this information belongs here: afaik the `ClusterLayout` public enum is an enumeration to add new clusters, and the orientation is something that is not used in all of them.

Plus, I don't find this enum useful to be exposed publicly. When would this be used? As it is right now is only used by the library to parse the `ComponentMeta` into a `ExpressibleByComponentMeta` stack config. I would keep this thing internal to the library somewhere else if you really need to use it (which maybe we don't really need it for now).